src/components/side_panel/side_panel/side_panel_store.ts

```diff
@@ -0,0 +1,36 @@
+import { SpreadsheetStore } from "../../../stores";
+
+interface SidePanelProps {
+  onCloseSidePanel?: () => void;
+  [key: string]: unknown;
```

Comment:
random question: what's the difference between putting `unknown` vs `any` here ?

Reply:
with `any`, typescript let's you do anything. `this.props.foo.bar` would compile. With `unknown`, typescript will prevent everything unless you explicitly check what it is before (you'd have to check `"foo" in this.props` before accessing it).

Most of the time, you want to use `unknown`